### PR TITLE
Introduce `manage_mysql_client` boolean to toggle whether or not to include mysql::client

### PR DESCRIPTION
**Pull request by walkleyn**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -170,6 +174,7 @@
   String $datadir = $proxysql::params::datadir,
   Stdlib::Filemode $datadir_mode = $proxysql::params::datadir_mode,
   Boolean $manage_selinux = true,
+  Boolean $manage_mysql_client = $proxysql::params::manage_mysql_client,
```
Comment (alexjfisher) [inline manifests/init.pp:177]: Could you just set the default `true` directly here and not use params.pp?

Comment (alexjfisher) [reply manifests/init.pp:177]: (Which reminds me, I should really finish https://github.com/voxpupuli/puppet-proxysql/pull/127)

Comment (walkleyn) [reply manifests/init.pp]: Done. I was unsure whether to follow the pattern like $manage_selinux or $manage_mycnf_file.

**Comment by alexjfisher** [comment]
@walkleyn Thanks!

**Pull request merged**